src/SDKs/SignalR/SignalR.Tests/SignalR.Tests.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>SignalR.Tests</PackageId>
+    <Description>SignalR.Tests Class Library</Description>
+    <AssemblyName>SignalR.Tests</AssemblyName>
+    <VersionPrefix>1.0.0.0</VersionPrefix>
```

Comment:
Replace this with `<Version>1.0.0</Version>`

Reply:
Updated the version.